packages/plugin-conversation/src/conversation.js

```diff
@@ -205,10 +287,20 @@ const Conversation = SparkPlugin.extend({
     });
   },
 
+  /**
+   * List activities for the specified conversation
+   * @param {Object} options
+   * @returns {Promise<Array<Activity>>}
+   */
   listActivities(options) {
```

Comment:
yea, maybe, I left it up to the api since they were both light wrappers around one of the _list methods